src/main/java/com/example/javawebcourse/web/CosmoCatController.java

```diff
@@ -0,0 +1,32 @@
+package com.example.javawebcourse.web;
+
+import com.example.javawebcourse.dto.CosmoCatDto;
+import com.example.javawebcourse.featuretoggle.FeatureToggle;
+import com.example.javawebcourse.featuretoggle.FeatureToggles;
+import com.example.javawebcourse.service.CosmoCatService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/cosmo-cats")
```

Comment:
If you want the /api to be prefixed to every endpoint of application, better to do it as follows:

application.properties
```
server.servlet.context-path=/api
```

I personally dont recommed it as it is just uninformative noise in route